pkg/cmd/server/admin/default_certs.go

```diff
@@ -145,7 +145,7 @@ func DefaultClusterAdminClientCertInfo(certDir string) ClientCertInfo {
 		},
 		UnqualifiedUser: "admin",
 		User:            "system:admin",
-		Groups:          sets.NewString(bootstrappolicy.ClusterAdminGroup, bootstrappolicy.MastersGroup),
+		Groups:          sets.NewString(bootstrappolicy.MastersGroup, bootstrappolicy.ClusterAdminGroup),
```

Comment:
A string set is an order independent map. You cannot depend on order in a go map. This could easily change in different go versions or in different invocations (map iteration order is randomized per invocation)

Reply:
It can never be simple :-)